tests/utils.py

```diff
@@ -359,6 +359,15 @@ def assert_rows_count(data: DataFrame, row_number: int):
             row_counter == row_number
         ), f"Expect {row_number} rows, Got {row_counter} instead"
 
+    @staticmethod
+    def assert_executed_with_query_tag(session: Session, query_tag: str) -> None:
+        query_details = session.sql(
```

Comment:
This will break in tests marked with localtest as session.sql does not work there yet.

Reply:
should I simply skip this test in local testing mode?

Reply:
updated test to skip the last step in this check